frontend/website/src/InclusiveSection.re

```diff
@@ -192,12 +193,14 @@ let make = _ => {
           link="/static/img/coda-figure.svg"
           dims=(15.125, 15.125)
           caption="Coda"
+          alt="Figure showing everyone participating in consensus, including all individual users of Coda."
```

Comment:
"all individual users" is plausible, but I think saying "including individual users" will be more accurate